src/cltk/alphabet/lat.py

```diff
@@ -1,12 +1,471 @@
+    text = text.replace(r"Ă", "A")
+    text = text.replace(r"à", "a")
+    text = text.replace(r"À", "A")
+    text = text.replace(r"â", "a")
+    text = text.replace(r"Â", "A")
+    text = text.replace(r"ä", "a")
+    text = text.replace(r"Ä", "A")
+    text = text.replace(r"é", "e")  # aegérrume
+    text = text.replace(r"è", "e")
+    text = text.replace(r"È", "E")
+    text = text.replace(r"é", "e")
+    text = text.replace(r"É", "E")
+    text = text.replace(r"ê", "e")
+    text = text.replace(r"Ê", "E")
+    text = text.replace(r"ë", "e")
+    text = text.replace(r"Ë", "E")
+    text = text.replace(r"ĭ", "i")  # ĭndignu
+    text = text.replace(r"î", "i")  # 'îs'
+    text = text.replace(r"í", "i")
+    text = text.replace(r"í", "i")
+    text = text.replace(r"î", "i")
+    text = text.replace(r"Î", "I")
+    text = text.replace(r"ï", "i")
+    text = text.replace(r"Ï", "I")
+    text = text.replace(r"ó", "o")  # óccidentem
+    text = text.replace(r"ô", "o")
+    text = text.replace(r"Ô", "O")
+    text = text.replace(r"ö", "o")
+    text = text.replace(r"Ö", "O")
+    text = text.replace(r"û", "u")
+    text = text.replace(r"Û", "U")
+    text = text.replace(r"ù", "u")
+    text = text.replace(r"Ù", "U")
+    text = text.replace(r"ü", "u")
+    text = text.replace(r"Ü", "U")
+    text = text.replace(r"ú", "u")  # frúges
+    text = text.replace(r"ÿ", "y")
+    text = text.replace(r"Ÿ", "Y")
+    text = text.replace(r"ç", "c")
+    text = text.replace(r"Ç", "C")
+    text = text.replace(r"ë", "e")
+    text = text.replace(r"Ë", "E")
+    text = text.replace(r"Ȳ", "Y")
+    text = text.replace(r"ȳ", "y")
+    return text
+
+
+def remove_macrons(text: str) -> str:
+    """
+    Remove macrons above vowels
+    :param text: text with macronized vowels
+    :return: clean text
+
+    >>> remove_macrons("canō")
+    'cano'
+
+    >>> remove_macrons("Īuliī")
+    'Iulii'
+
+    """
+    text = text.replace(r"ā", "a")
+    text = text.replace(r"Ā", "A")
+    text = text.replace(r"ē", "e")
+    text = text.replace(r"Ē", "E")
+    text = text.replace(r"ī", "i")
+    text = text.replace(r"Ī", "I")
+    text = text.replace(r"ō", "o")
+    text = text.replace(r"Ō", "O")
+    text = text.replace(r"ū", "u")
+    text = text.replace(r"Ū", "U")
+    return text
+
+
+def swallow_angle_brackets(text: str) -> str:
+    """
+    Disappear text in and surrounding an angle bracket
+    >>> text = " <O> mea dext<e>ra illa CICERO RUFO Quo<quo>. modo proficiscendum <in> tuis.  deesse HS <c> quae    metu <exagitatus>, furore   <es>set consilium  "
+    >>> swallow_angle_brackets(text)
+    'mea  illa CICERO RUFO  modo proficiscendum  tuis.  deesse HS  quae    metu  furore    consilium'
+
+    """
+    text = swallow(text, ANGLE_BRACKETS)
+    # There are occasionally some unmatched ANGLE_BRACKETS
+    text = text.replace("<", " ")
+    text = text.replace(">", " ")
+    return text
+
+
+def disappear_angle_brackets(text: str) -> str:
```

Comment:
Probably good for your texts, we should keep an eye out whether other people's text perhaps use such angle brackets for markup. Hopefully not